src/classes/OPP_CreateTribute_BATCH.cls

```diff
@@ -0,0 +1,129 @@
+                                            'FROM Opportunity';
+    
+    private String additionalWhereClause = null;
+
+    /*******************************************************************************************************
+    * @description no argument constructor for the class
+    */
+    public OPP_CreateTribute_BATCH() {
+
+    }
+
+    /*******************************************************************************************************
+    * @description constructor for the class
+    * @param additionalWhereClause - String allowing batch job to be invoked with alternative/additional constraints
+    */ 
+    public OPP_CreateTribute_BATCH(String additionalWhereClause) {
+        this.additionalWhereClause = additionalWhereClause;
+    }
+
+    /******************************************************************************************************
+    * @description Batch start method. Runs through all the Opportunities to be converted.
+    */
+    public Database.QueryLocator start(Database.BatchableContext info) {
+        String soqlQuery = ((additionalWhereClause != null) && (additionalWhereClause.trim().length() > 0)) ?
+                            OPP_QUERY + ' WHERE (' + additionalWhereClause + ')' : OPP_QUERY;
+
+        // Deliberately not trapping this potentially invalid query (since Where Clause might be invalid).
+        // This will display an error in the job widget used on the settings page, and send the person who 
+        // launched it a script error.  Decided this was best option with Nicolas Campbell
+        return Database.getQueryLocator(soqlQuery);
+    }
+
+    /*******************************************************************************************************
+    * @description Batch execute method.
+    * @return void.
+    ********************************************************************************************************/
+    public void execute(Database.BatchableContext BC, List<Opportunity> scope) {
+
+        List<Tribute__c> newTributes = new List<Tribute__c>();
+        for (Opportunity opp : scope) {
+            if (hasTributeInfo(opp)) {
+                newTributes.add(
+                    new Tribute__c(
+                        Opportunity__c = opp.Id,
+                        Tribute_Type__c = opp.Tribute_Type__c,
+                        Honoree_Contact__c = opp.Honoree_Contact__c,
+                        Honoree_Name__c = opp.Honoree_Name__c
+                    )
+                );
+
+                //clear the Opportunity fields
+                opp.Tribute_Type__c = null;
+                opp.Honoree_Contact__c = null;
+                opp.Honoree_Name__c = null;
+            }
+        }
+        
+        //We do not want to run through Opportunity or Tribute triggers when clearing Tribute Fields
+        //on the Opportunity
+        TDTM_TriggerHandler.disableTDTM = true;
+        insert newTributes;
+        update scope;
```

Comment:
This will still attempt to update every single Opportunity, even if it was not modified. I recommend creating a new List<Opportunity> for changed Opps that is updated at the end (if the collection isn't empty).